components/Search/SearchResultStats.tsx

```diff
@@ -0,0 +1,33 @@
+import { FunctionComponent } from "react"
+import { FormattedMessage } from "react-intl"
+
+type Props = {
+  numberOfHits: number,
+  numberOfFilters: number,
+  query: string | string[]
+}
+
+export const SearchResultStats: FunctionComponent<Props> = (
+  { numberOfHits, numberOfFilters, query },
+) => {
+  if (numberOfHits === 0 && numberOfFilters > 0) {
+    return (
+      <FormattedMessage
```

Comment:
For cases like this we should have a conditional component. Multiple return statements like this makes for a more imperative code.